src/Configuration.php

```diff
@@ -1279,9 +1279,7 @@ function (SplFileInfo $fileInfo): ?bool {
                 $composerPaths = (array) $path;
 
                 foreach ($composerPaths as $composerPath) {
-                    if ('' !== trim($composerPath)) {
-                        $paths[] = $composerPath;
-                    }
+                    $paths[] = '' !== trim($composerPath) ? $composerPath : $basePath;
```

Comment:
shouldn't the trimmed result be used here? If yes, updating the test to capture this case would be nice as well